Cannot find module 'url-extractor'

Please update README to handhold n00bs thru installing pre-requisites.
I am evidently missing some subtle aspect of installing gitdown/
You can reproduce the issue as follows:
```bash
$ sudo npm install gitdown -g
$ wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.23.2/install.sh | bash
$ nvm install iojs
$ npm install url-extractor
$ gitdown ./foo.gd --output-file ./foo.md
Error: Cannot find module 'url-extractor'
$ cat foo.gd 
{"gitdown": "include", "file": " "./wheeler.sh"}
cat wheeler.sh
#!/bin/sh
#
#       wheeler.sh
#       add wheel group to acl
#
if [ $UID -ne 0 ]; then
    echo "Superuser/root privileges are required to run this script."
    echo "e.g. \"sudo $0\""
    exit 1
fi
chgrp -R wheel $1
chmod -R g+rwx $1
```

Yes, that is one of the first things I always try...even did it again this morning to be certain and I still get the same error.

```shell
~/Projects/Testing/gitdown
📷  $ ls
debugging.md node_modules package.json
~/Projects/Testing/gitdown
📷  $ rm -fr ./node_modules
~/Projects/Testing/gitdown
📷  $ npm cache clear
~/Projects/Testing/gitdown
📷  $ npm install
npm WARN package.json gitdown@1.0.0 No description
npm WARN package.json gitdown@1.0.0 No repository field.
npm WARN package.json gitdown@1.0.0 No README data
~/Projects/Testing/gitdown
📷  $ git
git                         git-contrib                 git-delete-submodule        git-graft                   git-obliterate              git-repl                    git-touch                   gitexport
git-alias                   git-count                   git-delete-tag              git-ignore                  git-pull-request            git-setup                   git-undo                    github-email
git-archive-file            git-create-branch           git-effort                  git-info                    git-rebase-patch            git-shell                   git-upload-archive          gitk
git-back                    git-credential-osxkeychain  git-extras                  git-line-summary            git-receive-pack            git-show-tree               git-upload-pack             gittower
git-bug                     git-cvsserver               git-feature                 git-local-commits           git-refactor                git-squash                  git_branch
git-changelog               git-delete-branch           git-fresh-branch            git-lock                    git-release                 git-subtree                 git_info
git-commits-since           git-delete-merged-branches  git-gh-pages                git-missing                 git-rename-tag              git-summary                 gitdown
~/Projects/Testing/gitdown
📷  $ gitdown ./debugging.md --output-file ./debugging2.md
module.js:328
    throw err;
    ^

Error: Cannot find module 'url-extractor'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/bbohling/.nvm/versions/node/v4.2.6/lib/node_modules/gitdown/dist/gitdown.js:13:20)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/bbohling/.nvm/versions/node/v4.2.6/lib/node_modules/gitdown/dist/index.js:5:15)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at /Users/bbohling/.nvm/versions/node/v4.2.6/lib/node_modules/gitdown/dist/bin/index.js:74:19
    at Object.<anonymous> (/Users/bbohling/.nvm/versions/node/v4.2.6/lib/node_modules/gitdown/dist/bin/index.js:79:3)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
```

That is from MY package.json. I actually am using 2.4.7

```shell
~/Projects/Testing/gitdown
📷  $ npm -g ls --depth=0
/Users/bbohling/.nvm/versions/node/v4.2.6/lib
├──
├── check-dependencies@0.12.0
├── doctoc@1.0.0
├── gitdown@2.4.7
└── npm@2.14.12

~/Projects/Testing/gitdown
📷  $ gitdown ./debugging.md --output-file ./debugging2.md
module.js:328
    throw err;
    ^

Error: Cannot find module 'url-extractor'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/bbohling/.nvm/versions/node/v4.2.6/lib/node_modules/gitdown/dist/gitdown.js:13:20)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/bbohling/.nvm/versions/node/v4.2.6/lib/node_modules/gitdown/dist/index.js:5:15)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at /Users/bbohling/.nvm/versions/node/v4.2.6/lib/node_modules/gitdown/dist/bin/index.js:74:19
    at Object.<anonymous> (/Users/bbohling/.nvm/versions/node/v4.2.6/lib/node_modules/gitdown/dist/bin/index.js:79:3)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
```

--- a/src/gitdown.js
+++ b/src/gitdown.js
@@ -109,7 +109,7 @@ Gitdown.read = (input) => {
         const repositoryURL = gitinfo.compile({name: 'url'}, {gitdown}) + '/tree/' + gitinfo.compile({name: 'branch'}, {gitdown});
         const deadlink = Deadlink();
 
-        urls = URLExtractor.extract(markdown, URLExtractor.SOURCE_TYPE_MARKDOWN);
+        urls = URLExtractor.extractUrls(markdown, URLExtractor.SOURCE_TYPE_MARKDOWN);
 
         urls = urls.map((url) => {
             let resolvedUrl;

